tests/api/extensions/test_fields.py

```diff
@@ -15,7 +15,7 @@ def test_fields_filter_item():
     item = Item(
         id="test-fields-filter",
         geometry=Polygon.from_bounds(0, 0, 0, 0),
-        properties={"datetime": datetime.utcnow(), "foo": "foo", "bar": "bar"},
+        properties={"datetime": datetime.now(timezone.utc), "foo": "foo", "bar": "bar"},
```

Comment:
Sadly `datetime.utcnow()` is not timezone aware, so switch to `datetime.now(timezone.utc)` to get the same value, but timezone aware.